src/crypto/algorithms/base.ts

```diff
@@ -211,12 +214,11 @@ export class DecryptionError extends Error {
     public readonly detailedString: string;
 
     public constructor(
-        public readonly code: string,
```

Comment:
Is this a breaking change?

Reply:
Well first of all, this class (and indeed the whole file) is *supposed* to be internal, so anyone using it can keep both halves when it breaks.

In any case: It would only be a breaking change for things *constructing* a `DecryptionError`: anything reading `code` once it is created will implicitly cast the `DecryptionFailureCode` to a string. And I've not been able to find any code constructing a `DecryptionError` outside the js-sdk.